ci/base.yml

```diff
@@ -81,6 +81,7 @@ build_image_aarch64:
   variables:
     CSCS_ADDITIONAL_MOUNTS: '["/project/d121/icon4py/ci/testdata:$TEST_DATA_PATH"]'
     HPC_SDK_PATH: "/opt/nvidia/hpc_sdk/Linux_${ARCH}/22.11"
+    ICON4PY_NOX_UV_EXTRA_ARGS: "--extra=cuda11"
```

Comment:
True, the `cupy-cuda` python package installed in the system environment in Docker build is not available in the `uv` environment. We can remove it from the Docker file, I guess.